aws-mysql.yml

```diff
@@ -224,6 +225,34 @@ provision:
     field_name: maintenance_end_min
     details: The preferred maintenance end minute
     enum: *minute_enum
+  - field_name: deletion_protection
+    type: boolean
+    default: false
```

Comment:
**question**: Shouldn't this be true?

Reply:
The story says that the default should be `true`, but the PosgreSQL service offering defaults it to `false` intentionally: https://www.pivotaltracker.com/story/show/182758638

We chose to default to `false` in PostgreSQL because folks expect to be able to delete services with `cf delete-service` alone. There are situations where adding additional deletion protection can be good, but we decided that it should be opt-in for people who want a second layer of protection, as it's a frustrating workflow if you don't benefit from it. We can revisit that of course, but it doesn't make sense for MySQL to do one thing and PostgreSQL to do another.